tests/unit/User/AvatarUploaderTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Flarum\Tests\unit\User;
+
+use Flarum\User\AvatarUploader;
+use Flarum\User\User;
+use Illuminate\Contracts\Events\Dispatcher;
+use Illuminate\Database\Eloquent\Model;
+use Intervention\Image\ImageManagerStatic;
+use League\Flysystem\FilesystemInterface;
+use Mockery as m;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use PHPUnit\Framework\TestCase;
+
+class AvatarUploaderTest extends TestCase
+{
+    use MockeryPHPUnitIntegration;
```

Comment:
Thanks for researching this. I just [configured checks and included this for all unit tests on the `master` branch](https://github.com/flarum/core/commit/4f1adba38726f76ca15ad0cd43e7a84cc708d8a8) - now you just need to extend from the `Flarum\Tests\unit\TestCase` base class.